src/pages/wallet-overview/wallet-overview.spec.ts

```diff
@@ -347,7 +347,7 @@ describe("WalletOverviewPage", () => {
 
     comp.openTransferPage();
 
-    expect(navController.push).toHaveBeenCalledWith(TransferPage);
+    expect(navController.push).toHaveBeenCalledWith(TransferPage, jasmine.any(Object));
```

Comment:
It isn't important to check if the correct parameters are passed to the push method?